eng/tox/create_wheel_and_install.py

```diff
@@ -84,21 +97,13 @@ def cleanup_build_artifacts(build_folder):
         for wheel in discovered_wheels:
             # if the environment variable is set, that means that this is running where we
             # want to use the pre-built wheels
+            pkg_wheel_path =''
```

Comment:
`black` was ok with this? Legit surprising. Would have expected `pkg_wheel_path = ""`

Reply:
I had run black with only other file. i missed this one. Updated the code now using black for this file too.